src/sagemaker_container_support/environment.py

```diff
@@ -0,0 +1,607 @@
+
+    For more information about resourceconfig.json:
+https://docs.aws.amazon.com/sagemaker/latest/dg/your-algorithms-training-algo.html#your-algorithms-training-algo-running-container-dist-training
+
+    Returns:
+        resource_config (dict[string, object]): the contents from /opt/ml/input/config/resourceconfig.json.
+                        It has the following keys:
+                            - current_host: The name of the current container on the container network.
+                                For example, 'algo-1'.
+                            -  hosts: The list of names of all containers on the container network,
+                                sorted lexicographically. For example, `['algo-1', 'algo-2', 'algo-3']`
+                                for a three-node cluster.
+    """
+    return read_json(os.path.join(INPUT_CONFIG_PATH, RESOURCE_CONFIG_FILE))
+
+
+def read_input_data_config():  # type: () -> Dict[str, Dict[str, str]]
+    """Read the input data configuration from /opt/ml/input/config/inputdataconfig.json.
+
+        For more information about inpudataconfig.json:
+https://docs.aws.amazon.com/sagemaker/latest/dg/your-algorithms-training-algo.html#your-algorithms-training-algo-running-container-dist-training
+
+    Returns:
+            input_data_config (dict[string, object]): contents from /opt/ml/input/config/inputdataconfig.json.
+
+                                For example, suppose that you specify three data channels (train, evaluation, and
+                                validation) in your request. This dictionary will contain:
+
+                                {'train': {
+                                    'ContentType':  'trainingContentType',
+                                    'TrainingInputMode': 'File',
+                                    'S3DistributionType': 'FullyReplicated',
+                                    'RecordWrapperType': 'None'
+                                },
+                                'evaluation' : {
+                                    'ContentType': 'evalContentType',
+                                    'TrainingInputMode': 'File',
+                                    'S3DistributionType': 'FullyReplicated',
+                                    'RecordWrapperType': 'None'
+                                },
+                                'validation': {
+                                    'TrainingInputMode': 'File',
+                                    'S3DistributionType': 'FullyReplicated',
+                                    'RecordWrapperType': 'None'
+                                }}
+
+                                You can find more information about /opt/ml/input/config/inputdataconfig.json here:
+                                https://docs.aws.amazon.com/sagemaker/latest/dg/your-algorithms-training-algo.html#your-algorithms-training-algo-running-container-inputdataconfig
```

Comment:
I think you should move this link / comment into the docstring above Returns (and replace the link to https://docs.aws.amazon.com/sagemaker/latest/dg/your-algorithms-training-algo.html#your-algorithms-training-algo-running-container-dist-training with this link)

Reply:
Agreed! Thank you.